yapapi/payload/vm.py

```diff
@@ -110,6 +111,35 @@ async def manifest(
     min_cpu_threads: int = 1,
     capabilities: Optional[List[VmCaps]] = None,
 ) -> Package:
+    """
+    Build a reference to application payload.
```

Comment:
There will be links to this function's docs in `yagna-docs` https://github.com/golemfactory/yagna-docs/pull/130/files#diff-fdcad8ddcf749ce50b0b8272227b55535b80b791fb1fc7992693fe697b5d67f6R102